news/2020-07-16-summer-theme-park-2020-voting-open.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "Summer Theme Park 2020 Fanart Contest Voting Open"
+date: 2020-07-16 09:00:00 +0000
+---
+
+**10** votes, **116** entries — help us decide which fanart pieces are tall enough to ride the victory rollercoaster!
+
+![](https://assets.ppy.sh/contests/107/header.jpg)
+
+We went for something a little different this summer with the theme park bit, and our amazingly talented community artists have not disappointed with their performance!
+
+Head into the crowds today and pick out your favourite **10** entries, with a selection of the top-voted winners making it to the main menu as the Summer 2020 Seasonal backgrounds, and snagging their artists a bunch of osu!supporter and profile badge prizes in the process.
```

Comment:
I don't think "with" makes sense as a transition here, because the subject changes. maybe it just sounds odd to me but either way I think 2 sentences would be better